cssom-view/cssom-getClientRects-002.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <title>CSSOM View - GetClientRects().length is the same regardless source new lines</title>
+  <link rel="help" href="https://drafts.csswg.org/cssom-view-1/#dom-element-getclientrects">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+ </head>
+ <body>
+  <span id="single">
+   test test
+  </span><br/>
+  <span id="multiple">
+   test
+   test
+  </span>
+  <script>
+    test(function () {
+      const count = document.querySelector("#single").getClientRects().length;
+      const count2 = document.querySelector("#multiple").getClientRects().length;
+      assert_equals(count, 1, "count");
```

Comment:
The output in WebKit is


|                                    Subtest                                     | Results |                   Messages                   |
|--------------------------------------------------------------------------------|---------|----------------------------------------------|
|                                                                                | OK      |                                              |
| `CSSOM View - GetClientRects().length is the same regardless source new lines` | FAIL    | `assert_equals: count2 expected 1 but got 2` |

What `count2` refers to should be clear when one looks at the source.